src/countdown/util/countdownUtils.ts

```diff
@@ -0,0 +1,56 @@
+import {RemainingTimeBreakdown} from "../../core/utils/time/timeTypes";
+import {CountdownItem} from "../..";
```

Comment:
Can you check if it works without any import ?
```suggestion
```

Reply:
Yes, looks working but TSC gave some warning about `Cannot find name X` for `CountdownItem` and `CountdownProps`.